tests/api/test_bootstrap.py

```diff
@@ -27,6 +27,17 @@ def test_app_with_relative_path(simple_api_spec_dir, spec):
     assert get_bye.data == b'Goodbye jsantos'
 
 
+@pytest.mark.parametrize("spec", SPECS)
+def test_app_with_resolver(simple_api_spec_dir, spec):
+    from connexion.resolver import Resolver
+    resolver = Resolver()
+    app = App(__name__, port=5001,
+              specification_dir='..' / simple_api_spec_dir.relative_to(TEST_FOLDER),
+              resolver=resolver)
+    api = app.add_api(spec)
+    assert api.resolver == resolver
```

Comment:
Nitpicking, but don't we want to make sure ``api.resolver is resolver``?

Reply:
good call